DemoApp/index.js

```diff
@@ -25,13 +25,24 @@ app.use(cors());
 app.use(express.json());
 app.use(passport.initialize());
 
+const PORT = 5000
```

Comment:
Give priority to env variable PORT first and then if it doesn't exist then you can assign a default value here.